massa-ledger-exports/src/ledger_changes.rs

```diff
@@ -750,6 +750,44 @@ impl LedgerChanges {
         }
     }
 
+    pub fn get_keys(
```

Comment:
this duplicates code and also it won't work as expected: information on key deletion is lost.
I recommend removing this function.

Reply:
I think it is still there :)

Reply:
My bad. I just removed it!